Core/Games/KerbalSpaceProgram.cs

```diff
@@ -328,6 +332,94 @@ private string[] filterCmdLineArgs(string[] args, GameVersion installedVersion,
+            => nodes
+                .Select(node => node.For)
+                .Where(ident => !string.IsNullOrEmpty(ident))
+                .Concat(nodes.SelectMany(node => IdentifiersFromConfigNodes(node.Children)))
+                .Distinct();
+
+        /// <summary>
+        /// Find the identifiers associated with a .cfg file string
+        /// using ModuleManager's :FOR[identifier] pattern
+        /// </summary>
+        /// <param name="cfgContents">Path of the .cfg file relative to game root</param>
+        /// <returns>
+        /// Array of identifiers, if any found
+        /// </returns>
+        private static string[] CfgContentsToIdentifiers(GameInstance inst, string absolutePath, string cfgContents)
+            => KSPConfigParser.ConfigFile.Parse(cfgContents)
+                              .CaseOf(failure =>
+                                      {
+                                          log.InfoFormat("{0}:{1}:{2}: {3}",
+                                                         inst.ToRelativeGameDir(absolutePath),
+                                                         failure.State.Position.Line,
+                                                         failure.State.Position.Column,
+                                                         failure.Message);
+                                          return Enumerable.Empty<string>();
+                                      },
+                                      success => IdentifiersFromConfigNodes(success.Value))
+                              .ToArray();
+
+        /// <summary>
+        /// Find the identifiers associated with a manually installed .cfg file
+        /// using ModuleManager's :FOR[identifier] pattern
+        /// </summary>
+        /// <param name="absolutePath">Path of the .cfg file relative to game root</param>
+        /// <returns>
+        /// Array of identifiers, if any found
+        /// </returns>
+        private static string[] CfgPathToIdentifiers(GameInstance inst, string absolutePath)
+            => CfgContentsToIdentifiers(inst, absolutePath, File.ReadAllText(absolutePath));
+
+        public string[] IdentifiersFromFileName(GameInstance inst, string relativePath)
+            => relativePath.EndsWith(".dll", StringComparison.CurrentCultureIgnoreCase)
+                ? DllPathToIdentifiers(relativePath)
+                : relativePath.EndsWith(".cfg", StringComparison.CurrentCultureIgnoreCase)
+                    ? CfgPathToIdentifiers(inst, inst.ToAbsoluteGameDir(relativePath))
+                    : new string[] { };
+
+        public string[] IdentifiersFromFileContents(GameInstance inst, string relativePath, string contents)
+            => relativePath.EndsWith(".cfg", StringComparison.CurrentCultureIgnoreCase)
+                ? CfgContentsToIdentifiers(inst, relativePath, contents)
+                : new string[] { };
```

Comment:
Is this a left-over from development? It isn't called anywhere, and the functionality is also provided by `IdentifiersFromFileName`.